app/components/administrative-unit-status.js

```diff
@@ -0,0 +1,13 @@
+import Component from '@glimmer/component';
+
+export default class AdministrativeUnitStatusComponent extends Component {
+  get getStatusSkin() {
```

Comment:
nitpicky: `getStatusSkin` makes it sound like it's a function, but it's a getter. So I think simply using `statusSkin` might be clearer.